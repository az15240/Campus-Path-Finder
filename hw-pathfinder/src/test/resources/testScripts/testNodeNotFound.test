# Tests the special case for the Dijkstra function in which one or two given nodes are not found

CreateGraph g

AddNode g A
AddNode g B
AddNode g C

AddEdge g A B 1
AddEdge g B A 2
AddEdge g A C 3

ListNodes g

ListChildren g A

# One node not found

FindPath g A D

FindPath g A weird

FindPath g weird B


# Two nodes not found

FindPath g D E

FindPath g Stan Ley

FindPath g "" ""
