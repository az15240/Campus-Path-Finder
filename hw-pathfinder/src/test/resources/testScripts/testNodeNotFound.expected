# Tests the special case for the Dijkstra function in which one or two given nodes are not found

created graph g

added node A to g
added node B to g
added node C to g

added edge 1.000 from A to B in g
added edge 2.000 from B to A in g
added edge 3.000 from A to C in g

g contains: A B C

the children of A in g are: B(1.000) C(3.000)

# One node not found

unknown: D

unknown: weird

unknown: weird


# Two nodes not found

unknown: D
unknown: E

unknown: Stan
unknown: Ley

unknown: ""
unknown: ""
